tests/upgrades/test_errata.py

```diff
@@ -125,77 +125,60 @@ def test_pre_scenario_generate_errata_for_client(self, save_test_data):
             1. The content host is created
             2. errata count, erratum list will be generated to satellite client/content
                 host
-
         """
-        org = entities.Organization().create()
-        loc = entities.Location(organization=[org]).create()
-        environment = entities.LifecycleEnvironment(organization=org).search(
-            query={'search': 'name=Library'}
+        sat = katello_agent_client_for_upgrade.sat
+        rhel_client = katello_agent_client_for_upgrade.client
+        org = sat.api.Organization().create()
+        loc = sat.api.Location(organization=[org]).create()
+
+        sat.update_vm_host_location(rhel_client, loc.id)
+        environment = sat.api.LifecycleEnvironment(organization=org).search(
+            query={'search': f'name={constants.ENVIRONMENT}'}
         )[0]
-
-        product = entities.Product(organization=org).create()
-        custom_yum_repo = entities.Repository(
+        product = sat.api.Product(organization=org).create()
+        custom_yum_repo = sat.api.Repository(
             product=product, content_type='yum', url=settings.repos.yum_9.url
         ).create()
         product.sync()
-
-        tools_repo, rhel_repo = self._create_custom_rhel_tools_repos(product)
-        repolist = [custom_yum_repo, tools_repo, rhel_repo]
-        content_view = publish_content_view(org=org, repolist=repolist)
-        ak = entities.ActivationKey(
-            content_view=content_view, organization=org.id, environment=environment
+        synced_repos = self._create_custom_rhel_n_client_repos(sat, product, rhel_client)
+        repolist = [
+            custom_yum_repo,
+            *synced_repos,
+        ]
+        content_view = sat.publish_content_view(org, repolist)
+        ak = sat.api.ActivationKey(
+            content_view=content_view, organization=org, environment=environment
         ).create()
-        subscription = entities.Subscription(organization=org).search(
+        subscription = sat.api.Subscription(organization=org).search(
             query={'search': f'name={product.name}'}
         )[0]
-
         ak.add_subscriptions(data={'subscription_id': subscription.id})
```

Comment:
don't we required to run the `subscription manager refresh` content host as there is an update in the subscription, or does it is automatically taken care of?

Reply:
`refresh` added